boards/arm/hexiwear_k64/doc/hexiwear_k64.rst

```diff
@@ -95,39 +98,62 @@ Connections and IOs
-+-------+-----------------+---------------------------+
-| PTE24 | UART4_RX        | UART BT HCI               |
-+-------+-----------------+---------------------------+
-| PTE25 | UART4_TX        | UART BT HCI               |
-+-------+-----------------+---------------------------+
++-------+-----------------+----------------------------------+
+| Name  | Function        | Usage                            |
++=======+=================+==================================+
+| PTA13 | GPIO            | Docking station button/LED 2     |
++-------+-----------------+----------------------------------+
+| PTA14 | GPIO            | Docking station button/LED 3     |
++-------+-----------------+----------------------------------+
+| PTA29 | GPIO            | LDO_EN                           |
++-------+-----------------+----------------------------------+
+| PTB0  | I2C0_SCL        | I2C / MAX30101                   |
++-------+-----------------+----------------------------------+
+| PTB1  | I2C0_SDA        | I2C / MAX30101                   |
++-------+-----------------+----------------------------------+
+| PTB9  | GPIO            | Haptic motor feedback            |
++-------+-----------------+----------------------------------+
+| PTB12 | GPIO            | 3V3B EN                          |
++-------+-----------------+----------------------------------+
+| PTB16 | UART0_RX        | UART Console                     |
++-------+-----------------+----------------------------------+
+| PTB17 | UART0_TX        | UART Console                     |
++-------+-----------------+----------------------------------+
+| PTC8  | GPIO            | Red LED                          |
++-------+-----------------+----------------------------------+
+| PTC9  | GPIO            | Green LED                        |
++-------+-----------------+----------------------------------+
+| PTC10 | I2C1_SCL        | I2C / FXOS8700 / FXAS21002       |
++-------+-----------------+----------------------------------+
+| PTC11 | I2C1_SDA        | I2C / FXOS8700 / FXAS21002       |
++-------+-----------------+----------------------------------+
+| PTC18 | GPIO            | FXAS21002 INT2                   |
++-------+-----------------+----------------------------------+
+| PTD0  | GPIO            | Blue LED                         |
++-------+-----------------+----------------------------------+
+| PTD13 | GPIO            | FXOS8700 INT2                    |
++-------+-----------------+----------------------------------+
+| PTE24 | UART4_RX        | UART BT HCI                      |
++-------+-----------------+----------------------------------+
+| PTE25 | UART4_TX        | UART BT HCI                      |
++-------+-----------------+----------------------------------+
+
+
+GPIO lines for the Click board interfaces on the docking station are
+also defined.   Each interface (numbered 1-3) defines the following
+
+   .. code-block:: console
+      CLICKn_AN_{NAME, PIN}
```

Comment:
needs a blank line after the .. code-block line and before the first line of the code block contents (otherwise it complains about parameters to the codeblock tag)